metaci/build/migrations/0020_auto_20180907_1414.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.10 on 2018-09-07 14:14
+from __future__ import unicode_literals
```

Comment:
ideally squash the migrations before we push to prod?

Reply:
Squashed migration is in the package.  I never even knew about squashmigrations :)